src/index.html

```diff
@@ -4,9 +4,353 @@
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="main__container container">
+        <article class="container__card card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="image"
+          >
+
+          <h2 class="card__name-of-product">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars">
+            <div class="stars stars--light-4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__stars-reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <button class="card__button" data-qa="card-hover">buy</button>
+        </article>
+
+        <article class="container__card card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="image"
+          >
+
+          <h2 class="card__name-of-product">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars">
+            <div class="stars stars--light-4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__stars-reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <button class="card__button">buy</button>
+        </article>
+
+        <article class="container__card card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="image"
```

Comment:
It's a bad practice to use words like `image` in alt attribute, it's obvious that it's an image. Alt attribute should shortly describe what is pictured on this image. Also don't forget that this attribute is used for screenreaders